plugin-energy-estimation-tdp/README.md

```diff
@@ -0,0 +1,75 @@
+# energy estimation tdp plugin
+
+## Table of Contents <!-- omit in toc -->
+
+- [introduction](#Introduction)
+- [Energy estimation tdp plugin](#Energy-estimation-tdp-plugin)
+  - [How to use](#how-to-use)
+  - [Prepare your environment](#prepare-your-environment)
+  - [Configuration](#Configuration)
+
+## Introduction
+
+This plugin estimate the energy consumption for pods based on the TDP value of the machine where pods are running.
+The TDP (Thermal Design Power) is the maximum amount of heat generated by a computer compnent. 
+Refer to https://en.wikipedia.org/wiki/Thermal_design_power for more details.
+
+In the first version of the plugin, we consider only the TDP of the CPU.
+
+ This plugin requires the cgroupv2 input plugin (k8s) as it needs the input measurements of cgroupsv2.
+
+ The estimation calculation is done using the following formula:
+
+ energy estimation = cgroupv2_cpu_total_usage * nb_vcpu/nb_cpu * tdp / 1000 000 / polling_interval
```

Comment:
Can't we use latex to have a correctly displayed formula? Markdown support latex code blocks and/or simple formula

Reply:
Yes you are right , I found a solution but still some problems to display space, to be investigated.
Below the result:
![latex-md](https://github.com/user-attachments/assets/d752e9f2-9e77-41d8-b52c-0efcf6e09a60)

Reply:
As Titouan said, [it should work right into the markdown document](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions) :smiley: No need to use an external website (which would not work without internet, or after some time). Try to write the formula directly in the markdown source:
```md
$$latex here$$
```